packages/stylelint-config/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 <!-- Learn how to maintain this file at https://github.com/WordPress/gutenberg/tree/HEAD/packages#maintaining-changelogs. -->
 
 ## Unreleased
+-   Updated: `stylelint` to `15.1.0`.
```

Comment:
Let's ensure that these changes are recognized as breaking changes for the package:

```suggestion

### Breaking Changes

-   Updated: `stylelint` to `15.1.0`.
```